components/blitz/resources/omero/api/IPixels.ice

```diff
@@ -16,19 +16,190 @@ module omero {
+                 * Saves the specified rendering settings.
+                 *
+                 * @param rndSettings Rendering settings.
+                 **/
                 void saveRndSettings(omero::model::RenderingDef rndSettings) throws ServerError;
+
+                /**
+                 * Bit depth for a given pixel type.
+                 *
+                 * @param type Pixels type.
+                 * @return Bit depth in bits.
+                 **/
                 idempotent int getBitDepth(omero::model::PixelsType type) throws ServerError;
+
+                /**
+                 * Retrieves a particular enumeration for a given enumeration
+                 * class.
+                 *
+                 * @param enumClass Enumeration class.
+                 * @param value Enumeration string value.
+                 * @return Enumeration object.
+                 **/
                 idempotent omero::model::IObject getEnumeration(string enumClass, string value) throws ServerError;
+
+                /**
+                 * Retrieves the exhaustive list of enumerations for a given
+                 * enumeration class.
+                 *
+                 * @param enumClass Enumeration class.
+                 * @return List of all enumeration objects for the
+                 *         <i>klass</i>.
+                 **/
                 idempotent IObjectList getAllEnumerations(string enumClass) throws ServerError;
+
+                /**
+                 * Copies the metadata, and <b>only</b> the metadata linked to
+                 * a Pixels object into a new Pixels object of equal or
+                 * differing size across one or many of its three physical
+                 * dimensions or temporal dimension.
+                 * It is beyond the scope of this method to handle updates or
+                 * changes to the raw pixel data available through
+                 * {@link omero.api.RawPixelsStore} or to add
+                 * and link {@link omero.model.PlaneInfo} and/or other Pixels
+                 * set specific metadata.
+                 * It is also assumed that the caller wishes the pixels
+                 * dimensions and {@link omero.model.PixelsType} to remain the
+                 * same; changing these is outside the scope of this method.
+                 * <b>NOTE:</b> As {@link omero.model.Channel} objects are
+                 * only able to apply to a single set of Pixels any
+                 * annotations or linkage to these objects will be lost.
+                 *
+                 * @param pixelsId The source Pixels set id.
+                 * @param sizeX The new size across the X-axis.
+                 *              <code>null</code> if the copy should maintain
+                 *              the same size.
+                 * @param sizeY The new size across the Y-axis.                                 *              <code>null</code> if the copy should maintain
```

Comment:
Quite a lot of extraneous whitespace here that I can't see affects the rendering.